src/App.tsx

```diff
@@ -1,11 +1,16 @@
 import React from "react"
 
-import Navbar from "./components/Navbar/index"
-import { ReactComponent as Logo } from "./assets/profile.svg"
+import { BrowserRouter, Routes, Route } from "react-router-dom"
+import Navigation from "./components/Navigation"
+
 function App() {
   return (
     <div className="App">
-      <Navbar Logo={<Logo />} />
+      <BrowserRouter>
+        <Routes>
+          <Route path="*" element={<Navigation />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
You don't need to send this kind of change in your PRs. Make your tests in your local and don't push them inside the PRs.